### ISSUE TITLE
Opening Scripts by default in the SQL Editor

### ISSUE THREAD

**Issue by unlysted** [dbeaver/dbeaver#12021]
I have a problem with my install of DBeaver.
Originally when I clicked on files in the Projects Window, they opened in the SQL Editor.
Now when I click on files in the Projects Window, they open in the Projects Window instead of the SQL Editor.

I would like to know how to get them to open by default in the SQL Editor.

I have searched forums and the web but haven't found any solutions.
I have also asked other users, but they don't seem to know either.

**Comment by uslss** [comment]
@unlysted , which dbeaver version do you use? could you provide a screenshot or video with an issue?

**Comment by uslss** [comment]
there is no update on an issue for a long time.
If it is still actual for you - feel free to reopen it.

**Comment by unlysted** [comment]
My apologies.

I have been busy with projects and simply forgot to follow up with you.

I’ll send some examples later today.

Thank you for following up.

**Comment by unlysted** [comment]
I'm using version 21.0.4.202105021514.

I've attached two screenshots of the issue.

When I try to open a file in the projects window (Step 1.png) , that file
actually opens in the projects window (Step 2.png).

The application has not always done this.

Originally, clicking on a file in the Projects window opened the file in
the database editor window, which is to the right in both pictures.

This is annoying, and I can't seem to find a solution for this.

I have consulted several forums, asked other developers, and I've gone
through several updates of the application.

Thanks,

James
